next.config.js

```diff
@@ -0,0 +1,4 @@
+/** @type {import('next').NextConfig} */
```

Comment:
remove this comment, please
it looks extra

Reply:
fixed